### PR TITLE
Warn about backup redeem script

### LINKED ISSUE #2401: Add a warning to backup redeem scripts
If a VSP goes offline, it could result in permanently locked funds for some users that did not backup their redeem script.

The warning could be similar to the one in the docs.
https://docs.decred.org/wallets/decrediton/using-decrediton/#backup-redeem-script
Warning: Backup your redeem script
If your VSP goes down, you’ll need this script to vote or revoke tickets.

Perhaps add it to the "Configured VSP" box:
image

### PR DESCRIPTION
**Pull request by vctt94**
closes #2401

### CODE REVIEW

File: app/components/shared/ShowWarning.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const ShowWarning = ({ warn, className }) => (
-  <div className={"warning" + (className ? (" " + className) : "")}>{warn}</div>
+  <div className={"warning " + className}>{warn}</div>
```
Comment (matheusd) [inline app/components/shared/ShowWarning.js]: If  `className` is ever `undefined` or `null` the resulting `className` in the div ends up including a wrong value. Either revert the change or do a `["warning", className].join(" ")`.

File: app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/Form.js [hunk 2]
```diff
@@ -60,14 +63,38 @@ const PurchaseTicketsForm = ({
...
+          values= {{ link:
+            <ExternalLink href={"https://docs.decred.org/wallets/decrediton/using-decrediton/#backup-redeem-script"}>
+              <T id="purchase.ticket.decred.docs" m="Decred docs"/>
+            </ExternalLink> }}
+        />}/>
+        <div className="is-row backup-buttons-row-area">
+          <InvisibleConfirmModalButton
+            modalTitle={<T id="purchase.ticket.modal.title" m="Dismiss Button"/>}
+            modalContent={<T id="purchase.ticket.modal.desc"
+              m="Are you sure you want to dismiss this message? Make sure your redeem scripts are backed up" />}
```
Comment (matheusd) [inline app/components/views/TicketsPage/PurchaseTab/PurchaseTickets/Form.js]: Missing final period.

File: app/components/shared/ShowWarning.js (hunk 1 above)
Comment (vctt94) [reply app/components/shared/ShowWarning.js]: fixed
